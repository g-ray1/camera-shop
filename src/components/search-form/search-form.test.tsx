import { render } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import SearchForm from './search-form';

jest.mock('react-redux');

describe('SearchForm component', () => {
  it('Snapshot', () => {
    const view = render(<SearchForm />, {wrapper: BrowserRouter});

    expect(view).toMatchSnapshot();
  });
});
