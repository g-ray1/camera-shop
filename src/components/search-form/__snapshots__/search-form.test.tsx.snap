// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchForm component Snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="form-search"
      >
        <form>
          <label>
            <svg
              aria-hidden="true"
              class="form-search__icon"
              height="16"
              width="16"
            >
              <use
                xlink:href="#icon-lens"
              />
            </svg>
            <input
              autocomplete="off"
              class="form-search__input"
              placeholder="Поиск по сайту"
              type="text"
              value=""
            />
          </label>
        </form>
        <button
          class="form-search__reset"
          type="reset"
        >
          <svg
            aria-hidden="true"
            height="10"
            width="10"
          >
            <use
              xlink:href="#icon-close"
            />
          </svg>
          <span
            class="visually-hidden"
          >
            Сбросить поиск
          </span>
        </button>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="form-search"
    >
      <form>
        <label>
          <svg
            aria-hidden="true"
            class="form-search__icon"
            height="16"
            width="16"
          >
            <use
              xlink:href="#icon-lens"
            />
          </svg>
          <input
            autocomplete="off"
            class="form-search__input"
            placeholder="Поиск по сайту"
            type="text"
            value=""
          />
        </label>
      </form>
      <button
        class="form-search__reset"
        type="reset"
      >
        <svg
          aria-hidden="true"
          height="10"
          width="10"
        >
          <use
            xlink:href="#icon-close"
          />
        </svg>
        <span
          class="visually-hidden"
        >
          Сбросить поиск
        </span>
      </button>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
